.gitignore

```diff
@@ -14,7 +14,6 @@ MANIFEST
 *.egg
 *.egg-info
 dist
-build
```

Comment:
ah, right. stupid hfs. force updated to continuing ignoring `build`.

On Thu, Oct 1, 2015 at 12:11 PM, Russ Allbery <notifications@github.com>
wrote:

> In .gitignore
> <https://github.com/dropbox/grouper/pull/104#discussion_r40953907>:
>
> > @@ -14,7 +14,6 @@ MANIFEST
> >  *.egg
> >  *.egg-info
> >  dist
> > -build
>
> I think you still want to ignore this, don't you? As opposed to BUILD.
> (Case-insensitive file systems might have trouble there.) build is used
> by distutils, IIRC.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dropbox/grouper/pull/104/files#r40953907>.
>



-- 

herb